docs/get_started/intro.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+**P3G** is a tool designed to generate modern Python package project structures, leveraging Poetry for dependency management and integrating various development tools and best practices to help developers quickly create up-to-date Python projects with support for Python 3.7 and higher, automatic code formatting, type checking, security checks, pre-configured GitHub Actions, Docker support, and automated documentation generation.🚀
+
+## 💡 Features
+
+- ⏱️Faster formatter tool, automatic codestyle with ruff to replace black, isort and pyupgrade.
+- 🧰Everything is already set up for security checks, codestyle checks, code formatting, testing, linting, docker builds, etc with Makefile. More details in makefile-usage.
+- 🤖Files such as: LICENSE, CONTRIBUTING.md, CODE_OF_CONDUCT.md, and SECURITY.md are generated automatically.
+
+
+## 📗 Related Documentation
+
+- [Getting Started/Official Documentation]()
```

Comment:
these link cannot open

Reply:
You need to merge this pr before you can link to relevant documents

Reply:
why? [Doc](./you-doc-direction) is enough.